web/src/components/views/CreateProjectByPattern.jsx

```diff
@@ -261,15 +267,12 @@ export default class UploadByPattern extends React.Component {
           default: this.state.status
         },
         url: {
-          type: "string",
-          title: "Github URL",
-          format: "uri"
-        },
-        url_two: {
-          type: "string",
-          title: "Other URL",
-          format: "uri",
-          default: this.state.url
+          type: "array",
+          title: "URL",
+          items: {
+            type: "string"
+          },
```

Comment:
I'm gonna tackle this issue by the end of the week. There are too many more important issues under construction right now.